src/main/java/de/minebench/syncinv/SyncInv.java

```diff
@@ -286,7 +286,7 @@ public void onDisable() {
     }
 
     public void loadConfig() {
-        saveDefaultConfig();
+        getConfig().options().copyDefaults(true); // save newly added config options to disk
```

Comment:
Does this actually do anything on its own? I would assume this needs the config to be saved afterwards. (And you wouldn't want that on loading as that would overwrite existing values) Also with the removed saveDefaultConfig I would assume that it doesn't initially save the config at all.

I don't think it really matters that much that the keys are in the config though. If they are missing they will be pulled in from the file in the jar and if someone with a config which is missing them wants to edit it they can just get the ones they want from the default file in the jar or on github.

Reply:
just loading fist should do it since we load from jar if missing from disk anyways

Reply:
We could just use the config in jar, if we don't overwrite the defaults in `getLang()`.
In this case we just have to reload and save.